easybuild/easyblocks/q/qt.py

```diff
@@ -41,18 +42,32 @@ class EB_Qt(ConfigureMake):
     Support for building and installing Qt.
     """
 
+    @staticmethod
+    def extra_options():
+        extra_vars = {
+             'platform': [None, "Target platform to build for (e.g. linux-g++-64, linux-icc-64)", CUSTOM],
+        }
+        return ConfigureMake.extra_options(extra_vars)
+
     def configure_step(self):
         """Configure Qt using interactive `configure` script."""
 
         self.cfg.update('configopts', '-release')
 
+        platform = None
         comp_fam = self.toolchain.comp_family()
-        if comp_fam in [toolchain.GCC]:  #@UndefinedVariable
-            self.cfg.update('configopts', '-platform linux-g++-64')
+        if self.cfg['platform']:
+                platform = self.cfg['platform']
```

Comment:
@jgphpc the indentation is off here, each indent level should be 4 spaces (same below)